### ISSUE TITLE
[BITV] 9.1.4.3/3.1 - "Modified" column - The text content of the column does not meet the contrast requirements for text - it is 2.9/3.36:1. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36951]
- [x] F2V

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222223021-1867da98-50ba-4427-af38-7ec6b23bbcda.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v3-n1

</details>

- [x] 9.1.4.3/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v10-n1

**Comment by nimishavijay** [comment]
Agreed with @AndyScherzinger's proposal in the design chat, since sorting by column is supported we could just show all items in `--color-main-text` in the "Modified" as well as "Size" column :) 

The only colours that should be used for text are `--color-main-text` and `--color-text-maxcontrast`, everything else is either not accessible or not a variable AFAIK.

**Comment by skjnldsv** [comment]
I think it changed with files2vue because I remember using values with proper contrast.
Can you update this issue with master pls?

**Comment by Pytal** [comment]
F2V verified using `--color-main-text`
![image](https://github.com/nextcloud/server/assets/24800714/3985681d-36ed-412b-9046-06a45ddef665)
![image](https://github.com/nextcloud/server/assets/24800714/3337c9ec-aad0-4c09-acc9-1d659bc6437c)

**Comment by AndyScherzinger** [comment]
@Pytal your Screenshots shows the column header, not the actual column's cells' values, they also need to pickup that same color

**Comment by Pytal** [comment]
> @Pytal your Screenshots shows the column header, not the actual column's cells' values, they also need to pickup that same color

Re-verified :)

**Comment by skjnldsv** [comment]
Thank you @Pytal :bow: :hugs:
